ovs/dal/hybrids/vdisk.py

```diff
@@ -40,7 +40,7 @@ class VDisk(DataObject):
     """
     STATUSES = DataObject.enumerator('Status', ['HALTED', 'NON_RUNNING', 'RUNNING', 'UNKNOWN'])
 
-    VDISK_NAME_REGEX = '^[0-9a-zA-Z][\-_a-zA-Z0-9]{1,48}[a-zA-Z0-9]$'
+    VDISK_NAME_REGEX = '^[0-9a-zA-Z][\-_a-zA-Z0-9]{1,98}[a-zA-Z0-9]$'
```

Comment:
why limit this at all?

Reply:
The character limitation was put in place by the previous guys but never enforced due to a bug.
So as to why it should be limited... Force the customer to think about their names...? :p